src/service/private.overseas.entity.details.ts

```diff
@@ -22,7 +22,9 @@ export const getPrivateOeDetails = async (
     overseasEntityId,
   );
 
-  checkErrorResponse(req, response, overseasEntityId, transactionId, "overseas entity");
+  if (checkErrorResponse(req, response, overseasEntityId, transactionId, "overseas entity")) {
```

Comment:
Just thinking whether there's a more appropriate name for this
```suggestion
  if (responseHasError(req, response, overseasEntityId, transactionId, "overseas entity")) {
```

```suggestion
  if (hasErrorResponse(req, response, overseasEntityId, transactionId, "overseas entity")) {
```